### ISSUE TITLE
Nothing to import ?

### ISSUE THREAD

**Issue by hadashiA** [pCYSl5EDgo/create-unitypackage#1]
Hello,  I tried using this with [VContainer](https://github.com/hadashiA/VContainer) .

I had two problems.
1. Uploaded artifact named `.unitypackage.zip`.  (Not a .unitypackage)
2. The created package can open by Unity, but cannot be imported. 

Unity's import dialog says:
> Nothing to import!
> All assets from this package are already in your project.

🤔 

- Unity 2019.4.1f1 (macOS)
- workflow yaml
  - https://github.com/hadashiA/VContainer/blob/ku/unitypackage/.github/workflows/release.yaml#L9-L29
- Result
  - https://github.com/hadashiA/VContainer/runs/833573067?check_suite_focus=true

**Comment by pCYSl5EDgo** [comment]
Thank you for your Issue!

```
- name: Create meta list
  run: |
        echo "VContainer/Assets/VContainer.meta" > metaList
        find VContainer/Assets/VContainer/ -name \*.meta >> metaList
```

Due to the above instructions, Your workflow fails.
You should change directory to the root of your project.

```
- name: Create meta list
  run: |
        cd VContainer
        echo "Assets/VContainer.meta" > metaList
        find Assets/VContainer/ -name \*.meta >> metaList
```

And you should use `project-folder`option in create-unitypackage action.

**Comment by hadashiA** [comment]
Oh ok >< It has my mistake. Thanks for your teaching!! 

I tried to fix it.
But still have the same problem ... ? 😢 

Currently, extracted `pathname` file contents is starts with `Assets/`
`Assets/VContainer/Runtime/Unity/MonoInstaller.cs`

🤔 

- current workflow yaml
  - https://github.com/hadashiA/VContainer/blob/ku/unitypackage/.github/workflows/release.yaml#L9-L31
- New result
   - https://github.com/hadashiA/VContainer/runs/836297673?check_suite_focus=true

**Comment by hadashiA** [comment]
Hmm.. 
I extracted the unitypackage and the contents are probably fine.

This problem is probably caused by upload-artifact.

My latest build has following step.

```yaml
    - uses: actions/upload-artifact@v2
      with:
        name: unitypackage
        path: VContainer.unitypackage
```

As a result, upload-artifact is a `VContainer.unitypackage.zip`, not `Vcontainer.unitypakcage`. 
This zip is not what Unity is expecting.

Similarly, there were problems with here:

```yaml
    - uses: actions/upload-artifact@v2
      with:
        name: unitypackage
        path: tmp/
```

When you unzip unitypackage.zip, it's a folder. I have been able to upload the VContainer.unitypackage It is not.
